### PR TITLE
rcd: move webgui apart; option to disable browser

### LINKED ISSUE #3601: bug: rcserver not checking the existence of static files
#### What is the problem you are having with rclone?

rclone is accepting `--rc` as a global config to enable the rcserver. However, it is not checking if the web GUI exists or not and trying to serve it directly. This will cause exception if the web gui is not here.

#### What is your rclone version (output from `rclone version`)

rclone v1.49.5
- os/arch: darwin/amd64
- go version: go1.12.10

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

Minimum reproductive: `rclone config --rc --rc-web-gui`

Expected: Download the web gui tarball and unzip it

Exception: shows 404 instead.

#### Possible fix

Check if the GUI exists or not as `rclone gcd` does.

Please also refer to https://forum.rclone.org/t/beta-testing-webgui-for-rclone/11156/106 for a reproduction.

### PR DESCRIPTION
**Pull request by Yexiaoxing**
Fix #3601, #3785

#### What is the purpose of this change?

Enhance Web GUI.

- Move utils to a new file
- Make sure each time web gui is requested it checks release
- Add an option to disable opening browser
- minor enhancements

#### Was the change discussed in an issue or in the forum before?

#3601, #3785

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

**Comment by Yexiaoxing** [comment]
@Cnly Thanks for the review! Problems are fixed.

File: fs/rc/rcflags/rcflags.go [hunk 1]
```diff
@@ -21,7 +21,9 @@ func AddFlags(flagSet *pflag.FlagSet) {
 	flags.BoolVarP(flagSet, &Opt.Serve, "rc-serve", "", false, "Enable the serving of remote objects.")
 	flags.BoolVarP(flagSet, &Opt.NoAuth, "rc-no-auth", "", false, "Don't require auth for certain methods.")
 	flags.BoolVarP(flagSet, &Opt.WebUI, "rc-web-gui", "", false, "Launch WebGUI on localhost")
-	flags.BoolVarP(flagSet, &Opt.WebGUIUpdate, "rc-web-gui-update", "", false, "Update / Force update to latest version of web gui")
+	flags.BoolVarP(flagSet, &Opt.WebGUIUpdate, "rc-web-gui-update", "", true, "Check and update to latest version of web gui")
```
[the remark this answers is not in the document]
Comment (ncw) [reply fs/rc/rcflags/rcflags.go]: Removing an option will break backwards compatibility, so we can't do that at least without a deprecation period.

[the remark this answers is not in the document]
Comment (Yexiaoxing) [reply fs/rc/rcflags/rcflags.go]: Currently `rclone` will not check for update (as the default value is false). Should this behaviour change?

**Comment by ncw** [comment]
Thank you @Yexiaoxing for doing this and @Cnly for reviewing.

Apologies for the delay.

I'll merge this now.

**Comment by ncw** [comment]
PS @Yexiaoxing - Next time don't bother patching the README etc, I usually do this at release time only, just patch the `.md` docs - thanks.

**Comment by Yexiaoxing** [comment]
thanks for the reminder :)

**Pull request merged**